### ISSUE TITLE
Violation of Constraint: ContactNaamTelEmail

### ISSUE THREAD

**Issue by adam-ludera-intive** [kadaster/klic-win#77]
In many KLIC, the constraint on contactVoorzorgsmaatregel of AanduidingEisVoorzorgsmaatregel (ContactNaamTelEmail) is violated by providing empty Contact (the constraint implies that all three Contact fields must be not empty: naam, telefoon, email).

Example: KLIC 19G651470, AanduidingEisVoorzorgsmaatregel: nl.imkl-KL1101.AEVM_WMD_W_Leiding_32261_6632

**Comment by herman-vandenberg** [comment]
You are right, according to the IMKL Object-catalog, both name, telephone, and email must be filled ("notEmpty").
![image](https://user-images.githubusercontent.com/34309957/71589275-a1b48200-2b24-11ea-830b-88b20a6b70f1.png)
Apparently this is not currently being validated.
This will be a point of attention for the sector consultation (KGO).

**Comment by Marijnbiekart** [comment]
Speaking as a service provider: many operators explicitly do not want thier EV to be handled by email so in most cases the empty contact is intentional. Other placeholders like "aub bellen"  are common too.

**Comment by herman-vandenberg** [comment]
@Marijnbiekart 
Thank you for the addition. The sector (KGO/TCS) should give a definitive answer about this.

**Comment by FuatAkdeniz** [comment]
De TCS KLIC geeft aan dat dit relevante velden/waarden zijn voor het EV proces. We gaan ervan uit dat de velden nu al kwalitatief gevuld worden. Als TCS KLIC willen we dat hierop gevalideerd wordt: Of het gevuld is.
